tests/json/config-test.json

```diff
@@ -6,6 +6,5 @@
   "batchingIntervalInSeconds": 10,
   "observingIntervalInSeconds": 10,
   "maxConcurrentDownloads": 20,
-  "mongoDbConnectionString": "mongodb://localhost:27017/",
-  "transactionFeeMarkupPercentage": 10
```

Comment:
Sanity: don't forget to remove this from ION (and internal ION) repo also.

Reply:
Yes, I have an issue for that